controllers/watcher_controller.go

```diff
@@ -189,6 +193,19 @@ func (r *WatcherReconciler) Reconcile(ctx context.Context, req ctrl.Request) (re
 	_ = op
 	// end of TransportURL creation
 
+	// Create Keystone Service creation. The endpoint will be created by WatcherAPI
+	_, err = r.ensureKeystoneSvc(ctx, helper, instance, serviceLabels)
```

Comment:
given that we're adding a new condition `KeystoneServiceReadyCondition` shouldn't we move this call after we set `InputReadyCondition` to true so we can report more accurately what's wrong if there is any problem during this call?

Reply:
ok, I just saw the comment below, nvm

Reply:
I'm moving the ensureSecret to the Reconcile and moving the InputReadyCondition up with it.